src/framework/mlt_slices.h

```diff
@@ -38,4 +39,12 @@ extern int mlt_slices_count( mlt_slices ctx );
 
 extern mlt_slices mlt_slices_init_pool( int threads, int policy, int priority, const char* name );
 
+extern mlt_slices mlt_slices_get_global( mlt_schedule_policy policy );
```

Comment:
leaving only 3 functions:
```
extern void mlt_slices_run_normal( int jobs, mlt_slices_proc proc, void* cookie );
extern void mlt_slices_run_rr( int jobs, mlt_slices_proc proc, void* cookie );
extern void mlt_slices_run_fifo( int jobs, mlt_slices_proc proc, void* cookie );
```
very good approach...
i dont know why *mlt_slices_count_global* could be needed, but i have not objections against it.

why to rename: *pool_lock* -> *g_lock* - global functionality reusing existing mutex... twice mentioned in new function but 5 times in previous code...